department-of-veterans-affairs/va.gov-cms
# VACMS-6749 Update VHA Taxonomy Dashboard

## Description
Add "Required Vet Center service" and "Number of Vet Center facilities with service" to the health service dashboard

closes #6749 .

## Testing done
Navigated to view in question and was able to see the new columns and improved column structure. Exported csv and saw the new fields.

## Screenshots
![image](https://user-images.githubusercontent.com/8375335/138921715-a5c4ac2a-caa4-42d7-8f44-531b19e9b417.png)


## QA steps

As cms admin I am able to:
1. Navigate to `/vha-health-services` and see the view
2. Select a content type to filter the view to either VAMC services or Vet Center services
3. Confirm AC from ticket:

- [x] a new column added called "Required Vet Center service"
- [x] True is displayed as "Required". False is not displayed.
- [x] Sort by table headers (except concatenated columns)
- [x] CSV export matches new data displayed 

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
